includes/documents/abstract-wcpdf-order-document-methods.php

```diff
@@ -637,40 +637,38 @@ public function get_order_items() {
 	 * Get the tax rates/percentages for an item
 	 *
 	 * @param object $item order item
-	 * @param WC_Order|WC_Order_Refund $order WC_Order
+	 * @param WC_Abstract_Order $order WC_Order
 	 * @param bool $force_calculation force calculation of rates rather than retrieving from db
 	 *
-	 * @return string|float $tax_rates imploded list of tax rates
+	 * @return string $tax_rates imploded list of tax rates
 	 */
-	public function get_tax_rate( object $item, $order, bool $force_calculation = false ) {
+	public function get_tax_rate( object $item, object $order, bool $force_calculation = false ): string {
+		if ( apply_filters( 'wpo_wcpdf_calculate_tax_rate', false ) ) {
+			return '';
```

Comment:
Maybe `'-'`?

Reply:
You are returning it just below: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/722/files/71f8cd06f5db0f205acf55083a26e6e50e093563..dd560f14b7bd8c330534e39015a0e1f0c4204f24#diff-3c27d95148592338471e6af2989845953abad29560a97a4e73b989aaa9d1253fR653

Reply:
Yes, I assumed that the purpose of this filter was to bypass the method's calculation in favor of an alternative calculation method hence it should return nothing. This was just a thought because I couldn't find any examples of its use.